test/e2e/atlas/deployments_local_test.go

```diff
@@ -274,4 +274,40 @@ test   LOCAL   7.0.1     IDLE
 		a := assert.New(t)
 		a.Contains(o.String(), fmt.Sprintf("Index '%s' deleted", indexID))
 	})
+
+	t.Run("Pause Deployment", func(t *testing.T) {
```

Comment:
can you add e2e to deployments_atlas_test.go since atlas support was added here?

Reply:
yes, something to keep in mind is that pause and start are not available for the free tier so we will have to use m10 which will heavily increase the time for the test to complete

Reply:
interesting! I think we should still have this covered, but in benefit of time, I'm fine with doing a follow-up PR after this one since there's a lot of logic here already, WDYT?

Reply:
I have already the test ready for atlas, it is only one file more so I think we can add it here

Reply:
updated